test/test_devices.py

```diff
@@ -12,6 +13,9 @@ def setUpClass():
     xr.set_device_type('CPU')
     os.environ['CPU_NUM_DEVICES'] = '4'
 
+  def tearDown():
```

Comment:
oh is this being called after every test?

Reply:
Yeah, I decided to clear after every test in case we add more cases like me.